repair.py

```diff
@@ -67,6 +68,29 @@ def main():
             getItems = lambda media, childId: arr.getFiles(media=media, childId=childId) if args.mode == 'symlink' else arr.getHistory(media=media, childId=childId, includeGrandchildDetails=True)
             childrenIds = media.childrenIds if args.include_unmonitored else media.monitoredChildrenIds
 
+            # perform a "Safety check" to make sure that the torrents folder is mounted to prevent deleting everything in *arrs
+            # we do this inside of the loop because the mount could drop at any time; so, best to check with each iteration
+            mount_torrents_path = realdebrid['mountTorrentsPath']
+            if not os.path.exists(mount_torrents_path):
+                print(f"Path to mounted torrents does not exist: {mount_torrents_path}")
+                discordError(f"[{args.mode}] An error occurred while processing {media.title}: Path to mounted torrents does not exist", mount_torrents_path)
+                break

```

Comment:
My thinking was that if the mount is down, something serious is wrong.  By breaking, the script will still sleep for runIntervalSeconds and then start over. If the script is running in a docker container (likely), the only way I know of to fix the mount is to restart the container.  So, the script will likely be starting over anyway. If someone is running the script outside of Docker though, they could fix the mount while inflight.  So, I suppose sleeping and continuing would be a better option for them instead of starting from the beginning again.  Should I have it sleep for repairIntervalSeconds or create a new variable?

Reply:
Done.